completions/apt-get

```diff
@@ -34,11 +34,13 @@ _apt_get()
                     _filedir deb
                     return
                 elif [[ $cur == *=* ]]; then
-                    COMPREPLY=( $(compgen -W "$(\
-                        apt-cache --no-generate show "${cur%%=*}" 2>/dev/null |
-                        command sed -ne \
-                            's/^Version:[[:space:]]*\([^[:space:]]\)/\1/p')" \
-                        -- "${cur#*=}") )
+                    IFS='=' read -r package cur <<< "$cur"
+                    mapfile -t COMPREPLY < <(compgen -W "$( \
+                        while IFS=' |' read -r _ version _; do
+                            echo "$version"
+                        done < <(apt-cache madison "$package"))" \
```

Comment:
`--no-generate` and `2>/dev/null` were dropped, any particular reason? They seem as good ideas as before.

Reply:
The output from `apt-cache madison` does not consist of unneeded lines (not only of two as I mentioned above by mistake ;)) and is simpler to parse (it does not require any external tool like `sed`).

Right - the redirection of the error output was missing.

Reply:
But we still want `--no-generate` with it, no?